src/astgen

```diff
@@ -653,6 +663,21 @@ def read_types(filename, Nodes, prefix):
                                   "Aliased op" + str(n) + " is not defined")
                         else:
                             node.addOp(n, ident, *op[1:])
+                elif what in ("ptr"):
+                    ident, sep, kind = partitionAndStrip(rest, ":")
+                    ident = ident.strip()
+                    if not re.match(r'^m_(\w+)$', ident):
+                        error(
+                            lineno, "Malformed '@astgen " + what +
+                            "' directive (expecting '" + what +
+                            " := <identifier>': " + ident)
```

Comment:
Consider adding a hint in the error message that we need the `<identifier>` to start with a `m_` prefix.